pkg/cluster/describe.go

```diff
@@ -294,6 +295,15 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 	return nil
 }
 
+func printNodeInfo(replicasInfo string, securityGroups []string) string {
+	nodeStr := ""
+	nodeStr += fmt.Sprintf("\tReplicas: %s", replicasInfo)
+	if len(securityGroups) > 0 {
+		nodeStr += fmt.Sprintf("\n\tAWS Additional Security Group IDs: %s", strings.Join(securityGroups, ", "))
+	}
```

Comment:
if there will be security groups the multiple calls of `printNodeInfo` one right after the other will build the string
`"\tReplicas: %s"\tReplicas: %s"\tReplicas: %s` without creating new line. I would suggest to move the `\n` to the end of each string making: `\tReplicas: %s\n` and `\tAWS Additional Security Group IDs: %s\n`